rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/generic_object_type.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
```

Comment:
this doc should be before the class definition, jot the namespace

Reply:
You're right, getrector website did this wrong, will fix

Reply:
The fixture code is generated here https://github.com/rectorphp/getrector.org/tree/main/packages/github-magic-link/src/BodyFactory